test/Microsoft.Dnx.Tooling.FunctionalTests/DnuPackTests.cs

```diff
@@ -83,5 +83,30 @@ public void P2PDifferentFrameworks(DnxSdk sdk)
 
             TestUtils.CleanUpTestDir<DnuPackTests>(sdk);
         }
+
+        [Theory, TraceTest]
+        [MemberData(nameof(DnxSdks))]
+        public void DnuPack_ResourcesNoArgs(DnxSdk sdk)
+        {
+            // Arrange
+            var solution = TestUtils.GetSolution<DnuPackTests>(sdk, Path.Combine("ResourcesTestProjects", "ReadFromResources"));
+
+            var project = solution.GetProject("TestClassLibrary");
+            sdk.Dnu.Restore(project.ProjectDirectory).EnsureSuccess();
+
+            project = solution.GetProject("ReadFromResources");
+            sdk.Dnu.Restore(project.ProjectDirectory).EnsureSuccess();
+
+            // Act
+            var result = sdk.Dnu.Pack(project.ProjectDirectory);
+
+            // Assert
+            Assert.Equal(0, result.ExitCode);
+            Assert.True(Directory.Exists(Path.Combine(project.ProjectDirectory, "bin")));
```

Comment:
The result object has properties that allow you to find the output path without all of the `Path.Combine`